docker-compose.yaml

```
@@ -17,16 +15,27 @@ services:
   todo-app-dev:
     image: todo-app:dev
     container_name: todo-app-dev
-    #build:
-    #  context: /home/test/app/DevOps-Course-Starter        
     environment:
       - SECRET_KEY=secret-key
       - SECRET_APIKEY='YOUR_TRELLO_API_KEY'
       - SECRET_APITOKEN='YOUR_TRELLO_API_TOKEN'
       - BOARD_NAME='YOUR_DEV_TRELLO_BOARD_NAME'
     volumes:
-        #- .:/home/todo/app
         - /home/test/app/DevOps-Course-Starter:/home/todo/app
     ports:
       - 5001:5001 #dev port
-    restart: unless-stopped    
+    restart: unless-stopped
+
+  todo-app-test:
+    image: todo-app:test
+    container_name: todo-app-test
+    environment:
+      - SECRET_KEY=secret-key
+      - SECRET_APIKEY='YOUR_TRELLO_API_KEY'
+      - SECRET_APITOKEN='YOUR_TRELLO_API_TOKEN'
+      - BOARD_NAME='YOUR_DEV_TRELLO_BOARD_NAME'
+    volumes:
+        - /home/test/app/DevOps-Course-Starter:/home/todo/app
```

Note that this only works if every dev has this repo in the same folder as you, switching it to be relative will save devs fighting over the default!
```suggestion
        - .:/home/todo/app
```